work_queue/src/work_queue_submit_common

```diff
@@ -171,8 +176,14 @@ parse_arguments_common()
 	done
 
 	set_up_master_address $1 $2 $3 $4
+
+	if [ $specified_resources = 1 -a $SGE_WARNING = 1 ]
+	then
+		echo "Worker resources were manually specified. Remember to also describe your resources according to your local qsub system (e.g., -p '-pe smp 4'. See also the --sge-parameter configuration option.)" 1>&2
```

Comment:
Suggest that you explicitly name the "configure" script here, otherwise it may not be clear to the user.
Also add an item to the Makeflow manual explaining in more detail.